integration_tests/src/main/python/json_test.py

```diff
@@ -783,8 +783,6 @@ def test_from_json_struct_date_fallback_non_default_format(date_gen, date_format
     "\"" + optional_whitespace_regex + yyyy_start_0001 + optional_whitespace_regex + "\"",
     # "dd/MM/yyyy"
     "\"" + optional_whitespace_regex + "[0-9]{2}/[0-9]{2}/[1-8]{1}[0-9]{3}" + optional_whitespace_regex + "\"",
-    # special constant values
-    pytest.param("\"" + optional_whitespace_regex + "(now|today|tomorrow|epoch)" + optional_whitespace_regex + "\"", marks=pytest.mark.xfail(condition=is_before_spark_320(), reason="https://github.com/NVIDIA/spark-rapids/issues/9724")),
```

Comment:
This parameter should not have been deleted, just the xfail marker removed, e.g.:
```python
    # special constant values
    "\"" + optional_whitespace_regex + "(now|today|tomorrow|epoch)" + optional_whitespace_regex + "\"",
```